src/assets/style/form-item.css

```diff
@@ -2,7 +2,8 @@
 
 .ant-form-item {
   &-label {
-    @apply mb-2 leading-none;
```

Comment:
Is there any reason to remove `mb-2`?

Reply:
because of using margin bottom here is not correct way, should use padding bottom